community/kernel/src/test/java/org/neo4j/kernel/api/index/IndexProviderCompatibilityTestSuite.java

```diff
@@ -72,6 +73,9 @@ public void setup()
 
         public Compatibility( IndexProviderCompatibilityTestSuite testSuite, IndexDescriptor descriptor )
         {
+            this.pageCacheAndDependenciesRule = new PageCacheAndDependenciesRule( DefaultFileSystemRule::new,
+                    fs -> TestDirectory.testDirectory( null, fs, "-" + testSuite.getClass().getSimpleName().substring( 0, 3 ) ) );
```

Comment:
3 characters might be a bit optimistic for future cases. If you just want it to run fine today 1 would suffice :)